oci/oci.go

```diff
@@ -95,6 +102,12 @@ type Runtime struct {
 	ctrStopTimeout           int64
 }
 
+// RuntimeHandler represents each item of the "crio.runtime.runtimes" TOML
+// config table.
+type RuntimeHandler struct {
+	RuntimePath string `toml:"runtime_path"`
+}
```

Comment:
:+1:  for using a nested struct. I think this might come in handy at some point.